### PR TITLE
[CUBRIDMAN-7] manual for CBRD-23649 (dual table)

### PR DESCRIPTION
**Pull request by joohok**
http://jira.cubrid.org/browse/CUBRIDMAN-7

English manual for dual class
Description of dual class is inserted into system catalog class section. 
- definition
- usage
- example

### CODE REVIEW

File: en/sql/catalog.rst [hunk 1]
```diff
@@ -882,6 +882,47 @@ A table that stores the progress status every time the **applylogdb** utility ap
...
+
++--------------------+---------------+----------------------------------------------------------+
+|   Attribute Name   |   Data Type   |   Description                                            |
++====================+===============+==========================================================+
+| dummy              | VARCHAR(1)    | Value used for dummy purpose only                        |
++--------------------+---------------+----------------------------------------------------------+
+
+The following example shows the result which ran the query that select pseudo column after inputting ";plan detail" or "SET OPTIMIZATION LEVEL 513;" in CSQL (:ref:`viewing-query-plan`).This shows the dual class is referenced automatically even if there is no FROM clause.
```
Comment (charischau) [inline en/sql/catalog.rst]: 1. ;plan--> ; plan (adding space between ';' and 'plan')

2. .This -->. This (adding space between '.' and 'This')

Comment (joohok) [reply en/sql/catalog.rst]: 1. ";plan" is a kind of command in CSQL, so I think it doesn't need space between ';' and 'plan' 
2. I will fix it thanks

**Comment by hornetmj** [comment]
@JoohoK Please review whether the added dual table is included in the result of the manual example.

File: en/sql/catalog.rst [hunk 2]
```diff
@@ -984,6 +1025,7 @@ The following example shows how to retrieve system classes that can be accessed
       'db_root'
```
Comment (joohok) [inline en/sql/catalog.rst:1025]: 'db_ha_apply_info' class need to be added. 
I think it is required to be covered in another issue.

**Review by charischau — APPROVED** [review]

**Review by wrlawodms — APPROVED** [review]

**Review by hornetmj — APPROVED** [review]

**Pull request merged**